src/status_im2/contexts/quo_preview/loaders/skeleton_list.cljs

```diff
@@ -17,19 +19,41 @@
                :value "Messages"}]}
    {:label "Blur?"
     :key   :blur?
+    :type  :boolean}
+   {:label "Animated?"
+    :key   :animated?
     :type  :boolean}])
 
 (defn cool-preview
```

Comment:
@FFFra, consider removing this copy & paste code to set up the preview. I merged yesterday a PR changing a few dozen previews, so for new code it would be good to do the same. For an example, see `status-im2.contexts.quo-preview.community.discover-card`.

Reply:
@FFFra, this is basically how the namespace can be written now, with less boilerplate:

```clojure
(ns status-im2.contexts.quo-preview.loaders.skeleton-list
  (:require [quo2.core :as quo]
            [reagent.core :as reagent]
            [status-im2.contexts.quo-preview.preview :as preview]))

(def descriptor
  [{:key     :content
    :type    :select
    :options [{:key :list-items}
              {:key :notifications}
              {:key :messages}]}
   {:key :blur? :type :boolean}
   {:key :animated? :type :boolean}])

(defn view ; remember to update the main namespace importing this var
  []
  (let [state (reagent/atom {:content       :messages
                             :blur?         false
                             :animated?     true
                             :parent-height 600})]
    (fn []
      [preview/preview-container {:state state :descriptor descriptor}
       [quo/skeleton-list @state]])))
```